shared/js/project.js

```diff
@@ -234,11 +251,17 @@ var projects = module.exports = {
     current.level = this.appToProjectUrl();
 
     if (channelId && current.isOwner) {
+      current.migratedToS3 = true;
```

Comment:
If you don't want to extract something into appOptions.project.migrateToS3, it would still be better to extract a helper function shouldMigrateToS3, so there is one boolean function we have to modify in order to migrate more levels.